book/src/drivers.md

```diff
@@ -48,6 +48,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Dothill-CSI](https://github.com/enix/dothill-csi) | `dothill.csi.enix.io` | v1.3 | Generic CSI plugin supporting [Seagate AssuredSan](https://www.seagate.com/fr/fr/support/dothill-san/assuredsan-pro-5000-series/) appliances such as [HPE MSA](https://www.hpe.com/us/en/storage/flash-hybrid.html), [Dell EMC PowerVault ME4](https://www.dell.com/fr-fr/work/shop/productdetailstxn/powervault-me4-series) and others ... | Persistent | Read/Write Single Node | Yes | Snapshot, Expansion
 [Ember CSI](https://ember-csi.io) | `[x].ember-csi.io` | v0.2, v0.3, v1.0 | Multi-vendor CSI plugin supporting over 80 Drivers to provide block and mount storage to Container Orchestration systems. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
 [Excelero NVMesh](https://github.com/Excelero/nvmesh-csi-driver) | `nvmesh-csi.excelero.com` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for Excelero NVMesh | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion
+[Exoscale CSI](https://github.com/exoscale/exoscale-csi-driver) | `csi.exoscale.com` | v1.x | A Container Storage Interface (CSI) Driver for Exoscale Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Topology
```

Comment:
Please change v1.x to a CSI spec version that your driver supports.

Reply:
I see here it is v1.8: https://github.com/exoscale/exoscale-csi-driver/blob/main/go.mod#L6

Reply:
https://github.com/kubernetes-csi/docs/pull/586/commits/b3e95f73f46ba1fe04d6e982f62a9aebdfe0b697